content/installation/setup/auth/Configure-http-header.md

```diff
@@ -0,0 +1,13 @@
+<!--
+title: "Configure HTTP Header"
+description: "Configure Contrast to connect to HTTP Header"
+tags: "installation setup EOP system settings connect http header "
+-->
+Contrast supports certificate login through a proxy using a HTTP Header token. To enable HTTP Header login: 
```

Comment:
* I wouldn't call it a "token". The HTTP Header added by the trusted HTTPs proxy contains the user's unique username
* Consider using "authentication" instead of "login"
* "certificate login" is a bit vague. Let's call it "x509 client certificate authentication"
* the trusted proxy authenticates the user then sends the user's username to Contrast in an HTTP header